src/main/java/microsoft/exchange/webservices/data/ExchangeServiceBase.java

```diff
@@ -456,7 +456,15 @@ private void traceHttpResponseHeaders(TraceFlags traceType,
    * @return DateTime Returned date is always in UTC date.
    */
   protected Date convertUniversalDateTimeStringToDate(String dateString) {
-    String localTimeRegex = "^(.*)([+-]{1}\\d\\d:\\d\\d)$";
+	  return convertDateStringToDate(dateString, true);
+  }
+
+  protected Date convertStartDateToUnspecifiedDateTime(String value) {
+	  return convertDateStringToDate(value, false);
+  }
```

Comment:
I would suggest the two above methods being removed. Since the developer can be forced to decide if it is an UTC String or not (in my opinion). `convertDateStringToDate` may be renamed to `convertUniversalDateTimeStringToDate` again I think.